src/elements/content-uploader/UploadInput.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */ // Disabling linting for tabIndex on label, unknown why it's there
+import * as React from 'react';
+
+// https://stackoverflow.com/questions/72787050/typescript-upload-directory-property-directory-does-not-exist-on-type
+// Extend the InputHTMLAttributes interface to include the directory attribute
+declare module 'react' {
+    interface InputHTMLAttributes<T> extends React.HTMLAttributes<T> {
+        directory?: string;
+        webkitdirectory?: string;
+    }
+}
```

Comment:
should this exist in some global or types location? other upload inputs would encounter the same error too right?

Reply:
This is the only input we are doing this in, I didnt see a reason to let the rest of them know they have two more props